### PR TITLE
PEP8

### PR DESCRIPTION
**Pull request by alexpalms**
Implemented with VSCode, autopep8 and pycodestyle

### CODE REVIEW

File: diambraArena/diambraEnvLib/diambra_pb2.py [hunk 1]
```diff
@@ -12,12 +12,9 @@
 _sym_db = _symbol_database.Default()
 
 
-
```
Comment (discordianfish) [inline diambraArena/diambraEnvLib/diambra_pb2.py:15]: This file is autogenerated, so we shouldn't reformat it

File: diambraArena/diambraEnvLib/diambra_pb2_grpc.py [hunk 2]
```diff
@@ -84,123 +85,139 @@ def Close(self, request, context):
...
+# This class is part of an EXPERIMENTAL API.
+
 
- # This class is part of an EXPERIMENTAL API.
 class EnvServer(object):
     """The DIAMBRA service definition.
     """
 
     @staticmethod
     def EnvInit(request,
```
Comment (discordianfish) [inline diambraArena/diambraEnvLib/diambra_pb2_grpc.py:126]: PEP8 says:

> Function names should be lowercase, with words separated by underscores as necessary to improve readability.
https://peps.python.org/pep-0008/#function-and-variable-names

Any idea why autopep8 didn't fix this?

File: diambraArena/diambraEnvLib/diambra_pb2.py (hunk 1 above)
Comment (alexpalms) [reply diambraArena/diambraEnvLib/diambra_pb2.py:15]: you mean the whole file?

File: diambraArena/diambraEnvLib/diambra_pb2_grpc.py [hunk 3]
```diff
@@ -16,45 +16,46 @@ def __init__(self, channel):
...
+            request_serializer=diambra__pb2.Empty.SerializeToString,
+            response_deserializer=diambra__pb2.Empty.FromString,
+        )
 
 
 class EnvServerServicer(object):
     """The DIAMBRA service definition.
     """
 
     def EnvInit(self, request, context):
```
Comment (alexpalms) [inline diambraArena/diambraEnvLib/diambra_pb2_grpc.py:49]: @discordianfish these names are created automatically and inherited from the proto file. Can we keep them as they are?

File: diambra/arena/wrappers/arena_wrappers.py [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+    :return: (Gym Environment) the wrapped diambra environment
+    """
+
+    if no_op_max > 0:
+        env = NoopResetEnv(env, no_op_max=no_op_max)
+
+    if sticky_actions > 1:
+        env = StickyActionsEnv(env, sticky_actions=sticky_actions)
+
+    if hard_core:
```
Comment (discordianfish) [inline diambra/arena/wrappers/arena_wrappers.py:141]: I think just hardcore is better

**Review by discordianfish — APPROVED** [review]
LGTM

File: diambra/arena/wrappers/arena_wrappers.py (hunk 4 above)
Comment (alexpalms) [reply diambra/arena/wrappers/arena_wrappers.py:141]: ok I'll also rename that

**Pull request merged**